Monika After Story/game/zz_pianokeys.rpy

```diff
@@ -126,15 +126,15 @@ label mas_piano_songchoice:
         menu:
             m "Did you want to play a song or play on your own, [player]?{fast}"
             "Play a song.":
-                m "Which song?"
+                m "Which song?" nointeract
                 show monika at t21
                 call screen mas_gen_scrollable_menu(song_list, mas_piano_keys.MENU_AREA, mas_piano_keys.MENU_XALIGN, final_item)
                 show monika at t11
 
                 $ pnml = _return
 
                 # song selected
-                if pnml != "None":
+                if pnml is not None:
```

Comment:
If I select Nevermind, pnml actually gets set to a boolean value. Causes a traceback.
the [Return](https://www.renpy.org/doc/html/screen_actions.html#Return) screen action does not accept None as a value in screens.

```
[code]
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script-ch30.rpy", line 1492, in script call
    call call_next_event from _call_call_next_event_1
  File "game/event-handler.rpy", line 2532, in script call
    call expression event_label from _call_expression
  File "game/zz_games.rpy", line 172, in script call
    call mas_piano_start
  File "game/zz_pianokeys.rpy", line 145, in script
    if pnml.launch_label:
  File "game/zz_pianokeys.rpy", line 145, in <module>
    if pnml.launch_label:
AttributeError: 'bool' object has no attribute 'launch_label'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script-ch30.rpy", line 1492, in script call
    call call_next_event from _call_call_next_event_1
  File "game/event-handler.rpy", line 2532, in script call
    call expression event_label from _call_expression
  File "game/zz_games.rpy", line 172, in script call
    call mas_piano_start
  File "game/zz_pianokeys.rpy", line 145, in script
    if pnml.launch_label:
  File "C:\Users\potato\progs\renpy2222\renpy-6.99.12.4-sdk\renpy\ast.py", line 1656, in execute
    if renpy.python.py_eval(condition):
  File "C:\Users\potato\progs\renpy2222\renpy-6.99.12.4-sdk\renpy\python.py", line 1749, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\potato\progs\renpy2222\renpy-6.99.12.4-sdk\renpy\python.py", line 1743, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/zz_pianokeys.rpy", line 145, in <module>
    if pnml.launch_label:
AttributeError: 'bool' object has no attribute 'launch_label'

Windows-8-6.2.9200
Ren'Py 6.99.12.4.2187
Monika After Story 0.12.2
[/code]

```

Reply:
Ren'Pain